canonical/ubuntu.com
# Wd 12322 modify credentials shop to use the new products and coupons

## Done

- Credentials now uses coupons to make the products freely available to everyone instead of a 0 price product

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- visit /credentials/shop
    - Purchase an exam
    - Exam should be priced at 0 and should be purchased successfully
- visit /credentials/redeem
    - Redeem enter an activation key
    - See that the dropdown is populated with valid exam products
    - Redeem the key
        - Check if exam is redeemed and visible in `/credentials/your-exams`  

## Issue / Card

Fixes #WD-12322

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
